tests/parsecsv/tparsecsv.nim

```diff
@@ -0,0 +1,18 @@
+ discard """
```

Comment:
This test should be in the `stdlib` directory: https://github.com/nim-lang/Nim/tree/devel/tests/stdlib

The tests directory contains categories, so ``parsecsv`` is a bit too specific.